src/main/java/bio/terra/cli/serialization/userfacing/resource/UFGcpNotebook.java

```diff
@@ -62,6 +66,10 @@ public void print(String prefix) {
     OUT.println(prefix + "Instance id:       " + instanceId);
     OUT.println(prefix + "Instance name: " + (instanceName == null ? "(undefined)" : instanceName));
     OUT.println(prefix + "State:         " + (state == null ? "(undefined)" : state));
+    OUT.println(prefix + "Metadata:");
+    for (Map.Entry<String, String> entry : metadata.entrySet()) {
+      OUT.println("   " + entry.getKey() + ":" + entry.getValue());
```

Comment:
Put a space after the colon like ": ". Also try a tab to see what looks better (":\t").